test_scripts/Defects/7_1/885_ShowConstantTBT.lua

```diff
@@ -1,7 +1,7 @@
 ---------------------------------------------------------------------------------------------------
 -- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/885
 ---------------------------------------------------------------------------------------------------
--- Description: Check that SDL stops processing softButton events once a response is received for the RPC.
+-- Description: Check that SDL does not stop processing softButton events once a response is received for the RPC.
```

Comment:
I would add a link to `3829` issue along with the previous `885`

Reply:
@dboltovskyi Done in #150